infohub/settings.py

```diff
@@ -62,9 +62,12 @@
     'django.contrib.sites',
     'social_django',
     'firstaide',
-
+    'django_wysiwyg',
+    'ckeditor',
 )
 
+DJANGO_WYSIWYG_FLAVOR = "ckeditor"
+
```

Comment:
Please add this at the end, so that the newer library configurations can always be found in a pattern one after another. Its only a good practise.